specs/AgriFood/AgriCrop/example.jsonld

```diff
@@ -0,0 +1,76 @@
+{
+    "@context": [
+        "https://forge.etsi.org/gitlab/NGSI-LD/NGSI-LD/raw/master/coreContext/ngsi-ld-core-context.json",
```

Comment:
you can use 

http://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld

Reply:
Fixed in (https://github.com/Ficodes/dataModels/commit/23b9255ad972db1a60616447fd6099e57b12fa73)